python/ray/util/client/worker.py

```diff
@@ -684,7 +684,7 @@ def _server_init(self,
                         runtime_env, tmp_dir, logger=logger)
                     # Remove excludes, it isn't relevant after the upload step.
                     runtime_env.pop("excludes", None)
-                    job_config.set_runtime_env(runtime_env)
+                    job_config.set_runtime_env(runtime_env, validate=True)
```

Comment:
does this need to be optional or should we just always validate?

Reply:
Unfortunately for now it needs to be optional, because in some places we set a JobConfig before the working_dir and py_modules have been translated to URIs.  (validation expects them to already be URIs)